erpnext/accounts/doctype/pricing_rule/pricing_rule.py

```diff
@@ -291,6 +291,11 @@ def filter_pricing_rules(args, pricing_rules):
 		pricing_rules = filter(lambda x: (flt(stock_qty)>=flt(x.min_qty)
 			and (flt(stock_qty)<=x.max_qty if x.max_qty else True)), pricing_rules)
 
+		amt = args.get('qty') * args.get('conversion_factor', 1) * args.get('price_list_rate', 0)
+
+		pricing_rules = filter(lambda x: (flt(amt)>=flt(x.min_amt)
+			and (flt(amt)<=x.max_amt if x.max_amt else True)), pricing_rules)
```

Comment:
Seems you have added min_amt and max_amt using custom field in your local account and therefore it's not showing in the pricing_rule.json file, add these fields in the pricing rule doctype and push the pricing_rule.json file changes.